packaging/datadog-agent/source/setup_agent.sh

```diff
@@ -1,467 +1,515 @@
 #!/bin/sh
+# Bail on errors
+set -e
+# We shouldn't have unbounded vars
+set -u
+
+#######################################################################
+# SCRIPT KNOBS
+#######################################################################
+# Update for new releases, will pull this tag in the repo
+DEFAULT_AGENT_VERSION="5.6.3"
+# Pin pip version, in the past there was some buggy releases and get-pip.py
+# always pulls the latest version
+PIP_VERSION="6.0.8"
```

Comment:
might want to upgrade to 6.1.1 to get the latest bugfixes of the 6.x branch (that's also the version used in omnibus software)